### PR TITLE
1/2: Feat: menu bar

### PR DESCRIPTION
**Pull request by MathiasDuus**

### CODE REVIEW

**Comment by MathiasDuus** [comment]
[GUI - Menu bar](https://trello.com/c/PWUKXTVB/22-gui-menu-bar)

File: routes/api.php [hunk 1]
```diff
@@ -17,6 +17,7 @@
 /* Read */
 Route::group(['middleware' => ['auth:sanctum', 'ability:' . TokenEnum::READ->value]], function () {
     Route::get('/products/location', [ProductController::class, 'getProductsByUserLocation']);
+    Route::post('/roles/get_roles', [RoleController::class, 'getRoles']);
```
Comment (Setanna) [inline routes/api.php]: Send bearer token as header to use get instead of post request.

**Review by Setanna — CHANGES_REQUESTED** [review]

File: routes/api.php (hunk 1 above)
Comment (MathiasDuus) [reply routes/api.php]: Will do :)

File: app/Http/Controllers/RoleController.php [hunk 2]
```diff
@@ -86,12 +87,22 @@ public function update(RoleRequest $request, Role $role): RoleResource
...
+
+    /**
+     * Get current user roles
+     *
+     * @return JsonResponse
+     */
+    public function getRoles(): JsonResponse
+    {
+        return response()->json(auth()->user()->role);
+    }
```
Comment (Setanna) [inline app/Http/Controllers/RoleController.php]: Should it use the auth('sanctum') guard?
Additionally should it have a auth('sanctum)->check() to check if user is logged in

**Review by Setanna — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/RoleController.php (hunk 2 above)
Comment (MathiasDuus) [reply app/Http/Controllers/RoleController.php]: I see no reason to not

**Review by Setanna — APPROVED** [review]

**Pull request merged**